app/helpers/hostgroups_helper.rb

```diff
@@ -11,6 +11,14 @@ def warning_message(group)
     msg.join("\n")
   end
 
+  def editor_warnings
+    return unless @warnings
+
+    @warnings.map do |warning|
+      content_tag :div, warning, :class => ['alert', 'alert-warning']
```

Comment:
I'd suggest using `alert(:class => 'alert-warning'...` instead. That sets the icon too.